magda-web-client/package.json

```diff
@@ -86,5 +87,8 @@
     "docker": {
       "include": "node_modules build"
     }
+  },
+  "dependencies": {
+    "history": "^4.7.2"
```

Comment:
This is only needed in the webpack build, right?  So it can go in devDependencies.